contrib/Docker_Grafana_Blueflood/docker-entrypoint.sh

```diff
@@ -37,6 +37,8 @@ blueflood:
   apikey: $RAX_APIKEY                     
   authentication_module: rax_auth
   authentication_class: BluefloodAuth
+  enable_submetrics: true
+  submetric_aliases: { "_sum": 'sum' }
```

Comment:
@ChandraAddala right. This PR was more narrowly focused on the uptime reporting. I think with Vinny's research on StatsD metrics support we can turn on the rest of the submetrics in another PR.